worker/src/Worker.cpp

```diff
@@ -205,6 +206,7 @@ RTC::Router* Worker::GetRouterFromInternal(json& internal) const
 inline void Worker::OnChannelRequest(Channel::ChannelSocket* /*channel*/, Channel::ChannelRequest* request)
 {
 	MS_TRACE();
+	fprintf(stderr, "ENTERED OnChannelRequest: Channel request received \n");
```

Comment:
Let's use MS_WARN_TAG_STD instead b/c logging macros are already defined at this point

Reply:
Will do!